src/test/java/com/next/step/step3/util/RandomNumberGeneratorTest.java

```diff
@@ -0,0 +1,20 @@
+package com.next.step.step3.util;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RandomNumberGeneratorTest {
+
+    private static final int MIN_NUM = 0;
+
+    private static final int MAX_NUM = 9;
+
+    @Test
```

Comment:
```suggestion
    @RepeatedTest(10)
```
- 이런 학습테스트의 경우, 반복 테스트를 적용해보는건 어떨까요?

Reply:
맞네요ㅠ ㅎㅎ 리뷰 반영할게요~